src/dotnet-scaffolding/dotnet-scaffold-aspnet/Templates/EfController/ApiEfController.cs

```diff
@@ -33,6 +33,9 @@ public virtual string TransformText()
     string dbContextName = Model.DbContextInfo.DbContextClassName;
     string entitySetName = Model.DbContextInfo.EntitySetVariableName ?? modelName;
     string modelNamespace = Model.ModelInfo.ModelNamespace;
+    string primaryKeyName = Model.ModelInfo.PrimaryKeyName;
+    string primaryKeyNameLowerInv = primaryKeyName.ToLowerInvariant();
```

Comment:
I'm a little perplexed by all the ToLowerInvariant stuff - shouldn't it really be camel casing if these are used for variable names? Is the assumption that they're all one word (e.g. `id`)? Though I note that's not a change, since the existing code already has it that way (e.g. `modelNameLowerInvariant`).

Reply:
yep I think I have just been assuming its one word variable names, they could end up being 'moviekey' when it should be 'movieKey'. Will fix this